src/components/TodoContainer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+
+//  eslint-disable-next-line
```

Comment:
I think it will be better if there is no custom changes to linter config in order to meet project requirements.